CHANGELOG.rst

```diff
@@ -12,6 +12,17 @@ Change Log
 Unreleased
 ----------
 
+[9.0.0] - 2023-11-27
+
+Fixed
+~~~~~
+* **BREAKING CHANGE**: Fixes a bug for any service other than the identity service (LMS/CMS), where the session's local service user id would never match the JWT LMS user id when compared.
+
+  * The custom attribute jwt_auth_mismatch_session_lms_user_id was renamed to jwt_auth_mismatch_session_lms_user_id to make this more clear.
```

Comment:
I think the first copy here is supposed to be `jwt_auth_mismatch_session_user_id`

Reply:
Thanks. I created https://github.com/openedx/edx-drf-extensions/pull/411.